docs/content/config/advanced/kubernetes.md

```diff
@@ -4,513 +4,754 @@ title: 'Advanced | Kubernetes'
-kind: Deployment
+    ---
+    apiVersion: metallb.io/v1beta1
+    kind: IPAddressPool
 
-metadata:
-  name: mailserver
+    metadata:
+      name: mail
+      namespace: metallb-system
 
-# ...
     spec:
-      hostNetwork: true
+      addresses: [ <YOUR PUBLIC, DEDICATED IP IN CIDR NOTATION> ]
+      autoAssign: true
 
-    # ...
-      containers:
-        # ...
-          ports:
-            - name: smtp
-              containerPort: 25
-              hostPort: 25
-            - name: smtp-auth
-              containerPort: 587
-              hostPort: 587
-            - name: imap-secure
-              containerPort: 993
-              hostPort: 993
-        #  ...
-```
+    ---
+    apiVersion: metallb.io/v1beta1
+    kind: L2Advertisement
 
-With this approach,
+    metadata:
+      name: mail
+      namespace: metallb-system
 
-- it is not possible to access DMS via other cluster Nodes, only via the Node DMS was deployed at.
-- every Port within the Container is exposed on the Host side.
+    spec:
+      ipAddressPools: [ mailserver ]
+    ```
 
-### Proxy Port to Service via PROXY Protocol
+    **Advantages / Disadvantages**
 
-This way is ideologically the same as [using a proxy pod](#proxy-port-to-service), but instead of a separate proxy pod, you configure your ingress to proxy TCP traffic to the DMS pod using the PROXY protocol, which preserves the real client IP.
+    - :+1: simple
+    - :-1: requires a dedicated, publicly routable IP address
+    - :-1: limited to the node with the dedicated IP address
+    - :point_right: requires the setup of a load balancer
 
-#### Configure your Ingress
+=== "External-IP Service"
 
-With an [NGINX ingress controller][Kubernetes-nginx], set `externalTrafficPolicy: Local` for its service, and add the following to the TCP services config map (as described [here][Kubernetes-nginx-expose]):
+    **General**
 
-```yaml
-25:  "mailserver/mailserver:25::PROXY"
-465: "mailserver/mailserver:465::PROXY"
-587: "mailserver/mailserver:587::PROXY"
-993: "mailserver/mailserver:993::PROXY"
-```
+    !!! info
 
-!!! help "HAProxy"
-    With [HAProxy][dockerhub-haproxy], the configuration should look similar to the above. If you know what it actually looks like, add an example here. :smiley:
+        This approach only works when:
 
-#### Configure the Mailserver
+        1. You can dedicate a **publicly routable IP** address to the DMS configured `Service`.
+        2. The publicly routable IP is required to be dedicated to allow your mail server to have matching `A` and `PTR` records (_which other mail servers will use to verify trust when they receive mail sent from your DMS instance_).
 
-Then, configure both [Postfix][docs-postfix] and [Dovecot][docs-dovecot] to expect the PROXY protocol:
+    In this setup, you set up the DMS configured `Service` manually with an "[external IP][Kubernetes-network-external-ip]", providing the dedicated, publicly routable IP address yourself.
```

Comment:
Not sure what to do here exactly (the suggestions seems to already have been applied? I cannot find the texts though, weird...), but apply what you deem fitting :)

Reply:
> Not sure what to do here exactly (the suggestions seems to already have been applied?

I think I forgot this one sorry. I referred to a line you had since removed, and this change required me to edit the file content directly rather than a suggestion. I was out of executive functioning points by the time I pushed the larger changes, so I still need to add this suggestion, I'll take of it 👍